### ISSUE TITLE
Don't use transaction for DDL

### ISSUE THREAD

**Issue by rijkvanzanten** [directus/next#53]
MySQL always commits transactions in DDL, so using a transaction for schema modification doesn't help. 

`Transaction was implicitly committed, do not mix transactions and DDL with MySQL`.

Ideally, we rework this in a way that we can still rollback on error of schema modification, so we don't end up in a half-state of the DB. We should be able to do this by keeping track of what collections where inserted before, and using a `dropTableIfExists` when a later one fails to create

**Comment by WoLfulus** [comment]
Field modifications can be tricky though. I wasn't aware of this, we're still doing it in transactions on PHP side.

**Comment by rijkvanzanten** [comment]
Yeah.. Apparently basically any schema operation is committed right away:

https://dev.mysql.com/doc/refman/8.0/en/implicit-commit.html
